site/docs/foundations/spacing.mdx

```diff
@@ -61,73 +63,35 @@ When content directly following a header is a primary focus, for instance, the f
 | H3      | `--salt-spacing-200` | `--salt-spacing-100` |
 | H4      | `--salt-spacing-200` | None                 |
 
-### Non-hierarchical spacing
-
-When text following a header is a secondary focus, such as a section description intended as a supporting element to the section's content, spacing is a standard size regardless of the heading level. Keep the following in mind:
-
-- Headings followed by secondary focus text are consistently separated by `--salt-spacing-100`.
-- When a description follows a heading, the spacing is reduced to reinforce the relationship between the heading and description.
-
-<ImageSwitcher
-  images={[
-    { src: "/img/foundations/spacing-nonhierarchical.svg", alt: "" },
-    { src: "/img/foundations/spacing-nonhierarchical-annotated.svg", alt: "" },
-  ]}
-  label="Show spacing"
-/>
-
-| Heading | Space above          | Space below          |
-| ------- | -------------------- | -------------------- |
-| H1      | `--salt-spacing-200` | `--salt-spacing-100` |
-| H2      | `--salt-spacing-200` | `--salt-spacing-100` |
-| H3      | `--salt-spacing-200` | `--salt-spacing-100` |
-| H4      | `--salt-spacing-200` | `--salt-spacing-100` |
+### Container spacing
 
-### Headings and non-textual content
+A container represents an enclosed area with information, this is typically anything that is in a div in code. Spacing in containers varies depending on the container size.
```

Comment:
div should be `div` then you could remove the "in code" as anything in backticks is inline code.